src/models/oryp7/repairs.md

```diff
@@ -0,0 +1,200 @@
+
+Many components in your Oryx Pro can be upgraded or replaced as necessary. This page uses photos of the 17" model, but screw counts, general component locations, and other details are the same on the 15" model unless otherwise noted.
+
+- [Removing the bottom cover](#removing-the-bottom-cover)
+- [Replacing the RAM](#replacing-the-ram)
+- [Replacing an M.2/NVMe SSD](#replacing-an-m2nvme-ssd)
+- [Replacing the WiFi/Bluetooth module](#replacing-the-wireless-card)
+- [Replacing the battery](#replacing-the-battery)
+- [Replacing the CMOS battery](#replacing-the-cmos-battery)
+- [Replacing the fans/heatsink/thermal paste](#replacing-the-cooling-system)
+- [Replacing the keyboard](#replacing-the-keyboard)
+
+## Removing the bottom cover:
+
+Removing the cover is required to access the internal components. Prior to removing the cover, ensure the AC power is unplugged and all peripherals (including SD cards and USB drives) are unplugged or removed from the system.
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 5 minutes  
+**Difficulty:** Easy <span style="color:green;">●</span>
+
+### Steps to remove the bottom cover:
+
+1. Place the Oryx Pro lid-side down.
+2. Remove the 13 bottom panel screws.
+    - On the 15" model, the two back corner screws are longer than the rest (highlighted red below.)
+    - On the 17" model, all screws are the same length.
+
+![Bottom panel screws](./img/bottom-panel-screws.jpg)
+
+3. Lift the bottom panel off of the case.
+
+## Replacing the RAM:
+
+The Oryx Pro 7 supports up to 64GB (2x32GB) of DDR4 SO-DIMMs running at 3200MHz. If you've purchased new RAM, need to replace your RAM, or are reseating your RAM, follow these steps.
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 10 minutes  
+**Difficulty:** Easy <span style="color:green;">●</span>  
+
+### Steps to replace the RAM:
+
+1. Follow the steps above to [remove the bottom cover](#removing-the-bottom-cover).
+2. Press the small tabs on both sides of the RAM simultaneously. The RAM should spring up to an angle.
+
+![RAM removal](./img/ram-slots.jpg)
+
+3. Remove the RAM from the slot.
+4. Insert the new RAM (or reseat the existing RAM) by placing it in the keyed slot and pressing down on the RAM until it clicks into place.
+
+## Replacing an M.2/NVMe SSD:
+
+This model supports up to two M.2 SSDs. Both M.2 slots are size 2280. The inner-most slot (closest to the battery) supports only PCIe NVMe Generation 3, and the outer-most slot (farthest away from the battery) supports either SATA III or PCIe NVMe Generation 3.
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 10 minutes  
+**Difficulty:** Easy <span style="color:green;">●</span>  
+
+### Steps to replace the M.2 drive:
+
+1. Follow the steps above to [remove the bottom cover](#removing-the-bottom-cover).
+2. Unscrew the retainer screw opposite the M.2 slot.
+
+![M.2 slots](./img/m2-slots.jpg)
```

Comment:
Since one port only works with M.2 NVMe drives can we highlight which port that is?

Reply:
I mean, they're labeled on the motherboard.

Reply:
We state at the top of this section:

> The inner-most slot (closest to the battery) supports only PCIe NVMe Generation 3, and the outer-most slot (farthest away from the battery) supports either SATA III or PCIe NVMe Generation 3.

Do you not think this description is sufficient? I can see complaints about color-blindness accessibility by relying on colors for this kind of information where there's one item of either color. Also keep in mind that this information is physically silkscreened onto the motherboard, that this is the exact same wording the oryp6 manual is currently using, and that addw2, darp7, and serw12 also currently identify the slots by position instead of color.

Reply:
Sorry I'm blind @jacobgkau that should be fine. @leviport this is more about the photos having both slots filled so we can't see those labels.

Reply:
My point was that when someone's installing a drive, they'll be able to read the labels on the motherboard. That seems like the only time knowing which is which would matter, right?

Reply:
That is correct. I think my point is that having the information is almost never bad so I just wanted to float that idea by you cool folks.

Reply:
Other then that everything else looks amazing as usual! <3

Reply:
@ahoneybun I know we resolved this yesterday, but I later thought of adding labels directly on the image like this. I think it's clearer than the colors. What do you think?